### PR TITLE
Responsive UI

### PR DESCRIPTION
**Pull request by levant47**

### CODE REVIEW

File: src/services/telegram/message_manager.py [hunk 1]
```diff
@@ -63,11 +63,12 @@ def set_bot_commands(self, bot_commands):
         with urlopen(set_bot_commands_url):
             pass
 
-    def answer_callback_query(self, callback_query_id):
+    def answer_callback_query(self, callback_query_id, text=None):
+        parameters = {"callback_query_id": str(callback_query_id)}
+        if text is not None:
+            parameters["text"] = text
```
Comment (TiredOfThisAll) [inline src/services/telegram/message_manager.py:69]: Should not we use more informative name for this variable?

Comment (TiredOfThisAll) [reply src/services/telegram/message_manager.py:69]: Like request_parameters?

File: src/bot/server/handlers/responsive_ui_handlers.py [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+from services.telegram.message_entities_builder import MessageEntitiesBuilder
+from services.telegram.message_manager import MAX_MESSAGE_LENGTH
+
+
+@callback_handler(constants.ButtonCallbackType.RESPONSIVE_UI_ADD_ME)
+@callback_handler(constants.ButtonCallbackType.RESPONSIVE_UI_REMOVE_ME)
+@callback_handler(constants.ButtonCallbackType.RESPONSIVE_UI_CROSS_OUT)
+@callback_handler(constants.ButtonCallbackType.RESPONSIVE_UI_UNCROSS_OUT)
+@callback_handler(constants.ButtonCallbackType.RESPONSIVE_UI_REFRESH)
+def handle_generic_responsive_ui_callback(handler_context, update_context):
```
Comment (levant47) [inline src/bot/server/handlers/responsive_ui_handlers.py:14]: We need to resolve performance issues before merging this in

Comment (TiredOfThisAll) [reply src/bot/server/handlers/responsive_ui_handlers.py:14]: Telegram resolved our problem, ty.

**Review by TiredOfThisAll — APPROVED** [review]
LGTM!

**Pull request merged**